templates/security/index.html

```diff
@@ -472,21 +474,51 @@ <h3>Extend your Ubuntu security maintenance</h3>
 
 <section class="p-strip is-deep" id="ua-support">
   <div class="row">
-    <div class="col-8">
+    <div class="col-7">
       <h2>Ubuntu Advantage</h2>
       <p class="p-heading--four">All of our security products are available for a one off fee or they are all included in our Ubuntu Advantage support&nbsp;packages.</p>
       <p>Ubuntu Advantage is the professional package of tools, technology and expertise from Canonical, helping organisations around the world get the most out of their Ubuntu deployments. It includes access to:</p>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Livepatch: automatic kernel security hotfixes without rebooting</li>
         <li class="p-list__item is-ticked">FIPS: certified cryptographic modules available for compliance requirements</li>
         <li class="p-list__item is-ticked">Landscape: the systems management tool for using Ubuntu at scale</li>
-        <li class="p-list__item is-ticked">Extended Security Maintenance: critical security updates after Ubuntu end-of-life</li>
+        <li class="p-list__item is-ticked">Extended Security Maintenance: critical security updates after Ubuntu end of the initial five year period</li>
         <li class="p-list__item is-ticked">Knowledge Base: a private archive of expert-written articles and tutorials</li>
         <li class="p-list__item is-ticked">Support: phone and web-based support at multiple service levels</li>
       </ul>
       <p><a href="/advantage" class="p-button--positive">Purchase Ubuntu Advantage</a></p>
       <p><a href="/support/contact-us" class="js-invoke-modal">Contact us about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
     </div>
+    <div class="col-5 col-start-large-8 u-vertically-center">
+      <div class="p-card">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            {{
+            image(
+            url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+            ) | safe
+            }}
+            <h4>Datasheet</h4>
+          </div>
+        </div>
+        <h3>Learn more about Ubuntu’s security</h3>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Detailed overview</li>
+          <li class="p-list__item is-ticked">Key benefits</li>
+        </ul>
+        <p>
+          <a class="p-button" href="https://assets.ubuntu.com/v1/4a8eed26-Livepatch-Datasheet_25.10.21.pdf"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'resource', 'eventAction' : 'Livepatch datasheet', 'eventLabel' : 'from ubuntu.com/security/livepatch', 'eventValue' : undefined });"><span
+              class="p-link--external">Download the datasheet</span></a>
```

Comment:
maybe we don't need this `p-link--external` class as it's from asset.ubuntu.com?

Reply:
I think we keep it as an external as assets.ubunut.com is still 'Ubuntu' but it is hosted at a different domain.